### PR TITLE
message page by cartaya

### PR DESCRIPTION
**Pull request by cmcartaya**

### CODE REVIEW

File: cartaya/message-page/resources/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	line-height: 1.38;
+    margin: 20px 0 0 0;
+    text-align: left;
+}
+
+.message_link {
+    color: #2196f3;
+}
+
+.message_fotter {
```
Comment (sarayp) [inline cartaya/message-page/resources/css/styles.css]: there is a typo (footer)

File: cartaya/message-page/resources/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    text-align: left;
+}
+
+.message_text {
+	color: #7b7b7b;
+	font-size: 16px;
+	line-height: 1.38;
+    margin: 20px 0 0 0;
+    text-align: left;
+}
```
Comment (sarayp) [inline cartaya/message-page/resources/css/styles.css]: indent above

File: cartaya/message-page/resources/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    align-items: flex-end;
+    color: #32373c;
+    display: flex;
+	font-size: 16px;
+	font-weight: 900;
+    justify-content: flex-start;
+	line-height: 1.25;
+    margin: 36px 0 0 0;
+	text-align: left;
+}
```
Comment (sarayp) [inline cartaya/message-page/resources/css/styles.css]: indent above

File: cartaya/message-page/resources/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+.message_title {
+	color: #32373c;
+	font-size: 36px;
+	font-weight: 900;
+	line-height: 1.33;
+    margin: 0;
+    text-align: left;
+}
```
Comment (sarayp) [inline cartaya/message-page/resources/css/styles.css]: indent above

File: cartaya/message-page/resources/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+.view-port {
+    align-items: center;
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+}
+
+.lato-font {
+    font-family: 'Camphor-Thin', 'Arial', sans-serif;
```
Comment (sarayp) [inline cartaya/message-page/resources/css/styles.css]: font-face is defining lato-regular. Here you are declaring camphor-thin...which one it is?

**Review by sarayp — CHANGES_REQUESTED** [review]

**Review by TahimiLeonBravo — APPROVED** [review]

**Review by sarayp — APPROVED** [review]

**Pull request merged**